chapters/5-snippet-information.md

```diff
@@ -326,22 +362,26 @@ In `tag:value` format multiple lines are delimited by `<text> .. </text>`.
 
 Example:
 
-    SnippetLicenseComments: <text>The concluded license was taken from package xyz, from which the snippet was copied into the current file.
-    The concluded license information was found in the COPYING.txt file in package xyz.</text>
+```text
+SnippetLicenseComments: <text>The concluded license was taken from package xyz, from which the snippet was copied into the current file.
+The concluded license information was found in the COPYING.txt file in package xyz.</text>
+```
 
 **5.7.6 ** RDF: Property `spdx:licenseComments` in class `spdx:Snippet`
```

Comment:
I think there's an extra space between `5.7.6` and the closing `**` tag -- it looks like that might be causing the rest of this section to render as bold? I'm guessing that space should be removed.